db/schema.rb

```diff
@@ -33,28 +33,6 @@
     t.index ["key"], name: "index_active_storage_blobs_on_key", unique: true
   end
 
-  create_table "audits", charset: "utf8", force: :cascade do |t|
```

Comment:
I had to double check these diffs (for `block_commitments`, `slots`, and `stake_boss_stake_accounts`), but they look good. There are no migration files for these so they don't belong in `schema.rb`. `rails db:migrate` should always produce the same result as `rails db:schema:load`